PoGo.NecroBot.Logic/Tasks/UseNearbyPokestopsTask.cs

```diff
@@ -111,22 +109,22 @@ private static bool SearchThresholdExceeds(ISession session)
             cancellationToken.ThrowIfCancellationRequested();
 
             //request map objects to referesh data. keep all fort in session
+            var pokeStops = await UpdateFortsData(session);
 
-            var mapObjectTupe = await GetPokeStops(session);
-            _pokestopList = mapObjectTupe.Item2;
```

Comment:
that is code in the pass, did you see how this code in the past>? check history you will see reason for all changes i made